angular-todo/e2e/centralized-login.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+
+test('centralized login angular', async ({ page }) => {
+  await page.goto('https://localhost:8443/home?centralLogin=true');
+
+  await page.getByRole('link', { name: 'Sign In', exact: true }).click();
+  await expect(page.getByText('Redirecting ...')).toBeVisible();
+
+  await page.getByLabel('User Name').fill('user01');
+  await page.getByLabel('Password').first().fill('Password1!');
+  await page.getByRole('button', { name: 'Next' }).click();
+
+  await page.waitForTimeout(3000);
+
+  await page.reload();
```

Comment:
@cerebrl hi jusitn. I had to add this as a temporary workaround because of a strang behaviour. When I log in successfully  the server stops responding and i have to reload the page manually for it to work. It only happens when running the tests, the very same code works perfectly fine if i run the app locally. 
Attached evidence.
cc @ryanbas21 we've been thorught this in one of our sync meet. Remember?
[scrnli_3_4_2024_5-15-40 PM.webm](https://github.com/ryanbas21/forgerock-sdk-samples/assets/156847686/4558e61c-6b6f-4515-bbd5-d5e9f8ed4a8e)

Reply:
Hmm, interesting. Okay. If you and Ryan have discussed this, then I'm good with it.